lib/zendesk_apps_support/validations/translations.rb

```diff
@@ -29,9 +29,23 @@ def locale_error(file, locale)
           ValidationError.new('translation.invalid_locale', file: file.relative_path)
         end
 
+        def format_error(file)
+          with_valid_json(file) do |json|
+            if json['app'] && json['app']['parameters']
+              parameters_node = json['app']['parameters']
+              parameters_node.keys.map do |param|
+                unless parameters_node[param].is_a?(Hash) && parameters_node[param].keys.include?('label')
+                  return ValidationError.new('translation.missing_required_key',
+                                             file: file.relative_path, missing_key: "#{param} must have 'label'")
```

Comment:
Will this work when this error message is translated? I reckon we'll probably need a new translation instead (e.g. `missing_label_key_for_parameter`) so that "must have 'label'" can actually get translated

Reply:
Oh yeah, good point. I wasn't 100% sure if we do have translations, other than English, for those keys. I could simplify things with the error message to make use of the same key. What do you think?

Reply:
Probably work asking Phil how good the error message needs to be for this. If it's a very rare problem then I think it's OK to use the current error message and simplify it. But if people are likely to get this wrong a lot then we should probably add another key :man_shrugging: